File: M4 Intermediate DSA/src/L13_Maths_Modular_Arthimetic_Introduction/Assignment1.java
package L13_Maths_Modular_Arthimetic_Introduction;

public class Assignment1 {

  public int solve(int A, int B, int C)  {
    long ans = 1;

    for (int num = 1; num <= B; num++) {
      ans = ((ans % C) * (A % C)) % C;
    }

    return (int) ans;
  }

}

File: M3_Intermediate_DSA/Introduction_to_Problem_Solving/assignment1.go
package main

func SolveA1(A int) int {
	var ans int

	for num := 1; num*num <= A; num++ {
		if A%num == 0 {
			if num*num == A {
				ans++
			} else {
				ans += 2
			}
		}
	}

	return ans
}
